src/js/components/Meter/__tests__/__snapshots__/Meter-test.tsx.snap

```diff
@@ -306,6 +364,7 @@ exports[`Meter default 1`] = `
   class="c0"
 >
   <svg
+    aria-label="bar meter with values  filled out of 100"
```

Comment:
Can we have a message that handles when there is no value?

Reply:
done